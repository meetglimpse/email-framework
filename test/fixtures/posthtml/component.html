<component
  src="test/stubs/components/component.html"
  text="Example"
  locals='{
    "foo": "bar"
  }'
>
Variable from page: [[ page.env ]]

  <component
    src="test/stubs/components/component.html"
    text="Nested component"
    locals='{
      "foo": "bar (nested)"
    }'
  >
Variable from page (nested): [[ page.env ]]
  </component>
</component>
