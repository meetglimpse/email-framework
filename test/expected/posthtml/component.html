Variable from attribute: Example

Variable from locals attribute: bar


Variable from page: maizzle-ci

  Variable from attribute: Nested component

Variable from locals attribute: bar (nested)


Variable from page (nested): maizzle-ci
